third_party/amd/lib/TritonAMDGPUToLLVM/BuiltinFuncToLLVM.cpp

```diff
@@ -100,7 +108,9 @@ class CallOpConversion : public mlir::RewritePattern {
     rewriter.setInsertionPointToEnd(currentBlock);
     rewriter.create<LLVM::CondBrOp>(loc, pred, trueBlock, falseBlock);
     rewriter.setInsertionPointToStart(trueBlock);
-    auto loadOp = rewriter.create<LLVM::LoadOp>(loc, elemTy, ptr);
+    auto loadOp =
+        nt ? rewriter.create<LLVM::LoadOp>(loc, elemTy, ptr, 0, false, true)
```

Comment:
Can we add some inline comments for the `false`/`true`? Like `/*isVolatile=*/false, /*isNonTemporal=*/true` so that it's clear what we are doing here.